name: "Jetracer"
scheme: distributed # centralised or distributed
N: 20
integrator_step: 0.3
<<<<<<< HEAD
track_choice: straight_line # 'straight_line', sinus, "t_junction", "merging"
=======
track_choice: straight_line # straight_line, sinus, t_junction, merging
>>>>>>> 4290d0f50a4d1c4554341639019e5076925347be
skip_solver_generation: false

add_slack: true
slack_value: 1.0e3

number_of_robots: 2
robot_1:
  start_x: -5.0
  start_y: 1.0
  start_theta: 0.0  # radians
robot_2:
<<<<<<< HEAD
  start_x: -1.0
  start_y: -3.0
  start_theta: 0.0  # radians
=======
  start_x: -3.0
  start_y: -1.0
  start_theta: 0.0 # radians
>>>>>>> 4290d0f50a4d1c4554341639019e5076925347be

verbose: false
debug_visuals: false
dart_simulator: false

solver_settings:
  iterations_centralised: 2
  iterations_distributed: 2 #1 #2  # iterations for distributed nmpc-ca combi
  solver_type: SQP_RTI # SQP_RTI (default) or SQP

control_frequency: 5.0
braking_acceleration: 0.4

polytopic:
  d_min: 0.2
  length: .5
  width: .5

contouring:
  num_segments: 5
  use_huber: false

weights:
  throttle: .1
  steering: .1
  lambda: 0.1  # centralised very sensitive on this weight
  s_dual: 0.1    # centralised very sensitive on this weight
  dmin_objective: .1
  velocity: 0.4
  reference_velocity: 0.4
  lag: 0.6
  contour: 0.6
  terminal_angle: 1.0 #0.0
  terminal_contour: 1.0 # 0.0
  terminal_lag: 1.0 # 0.0






